File: create_logo.php
<?php
/**
 * Script untuk membuat logo SYH Cleaning
 * Run dari terminal: php create_logo.php
 */

// Buat directory jika belum ada
$dir = __DIR__ . '/public/assets/images';
if (!is_dir($dir)) {
    mkdir($dir, 0755, true);
}

// Create image with GD library
$width = 200;
$height = 200;
$image = imagecreatetruecolor($width, $height);

// Define colors
$white = imagecolorallocate($image, 255, 255, 255);
$dark = imagecolorallocate($image, 31, 41, 55); // #1f2937
$light_gray = imagecolorallocate($image, 243, 244, 246); // #f3f4f6
$blue = imagecolorallocate($image, 59, 130, 246); // #3b82f6
$purple = imagecolorallocate($image, 124, 58, 237); // #7c3aed

// Fill with white background
imagefill($image, 0, 0, $white);

// Draw circle border
imagefilledellipse($image, 100, 100, 190, 190, $white);
imageellipse($image, 100, 100, 190, 190, $dark);
imageellipse($image, 100, 101, 190, 190, $dark);
imageellipse($image, 101, 100, 190, 190, $dark);

// Draw shoe icons
// Left shoe
imagefilledellipse($image, 55, 67, 30, 35, $light_gray);
imageellipse($image, 55, 67, 30, 35, $dark);
imageline($image, 40, 75, 70, 75, $dark);

// Right shoe  
imagefilledellipse($image, 95, 67, 30, 35, $light_gray);
imageellipse($image, 95, 67, 30, 35, $dark);
imageline($image, 80, 75, 110, 75, $dark);

// Draw star
$star_points = array(
    120, 52,  // top
    125, 65,  // top right
    138, 68,  // right
    128, 76,  // bottom right
    132, 89,  // bottom
    120, 82,  // bottom left
    108, 89,  // left
    112, 76,  // top left
    102, 68,  // left top
    115, 65   // top
);
imagefilledpolygon($image, $star_points, 5, $blue);

// Add text - SYH.CLEANING (using built-in font)
// Since built-in fonts are limited, we'll use imagestring
$text_color = $dark;
// Position text at bottom
imagestring($image, 2, 35, 155, 'SYH.CLEANING', $text_color);

// Add FREE DELIVERY text at bottom
imagestring($image, 1, 45, 175, 'FREE DELIVERY', $purple);

// Save image as PNG
$filename = $dir . '/logo.png';
if (imagepng($image, $filename)) {
    echo "✓ Logo created successfully!\n";
    echo "File: {$filename}\n";
    echo "Size: {$width}x{$height}px\n";
} else {
    echo "✗ Error creating logo image\n";
}

// Free up memory
imagedestroy($image);
?>

File: create_logo.py
#!/usr/bin/env python3
"""
Script untuk membuat logo SYH Cleaning
Run: python create_logo.py
"""

from PIL import Image, ImageDraw, ImageFont
import os

# Create directory if not exists
output_dir = 'public/assets/images'
if not os.path.exists(output_dir):
    os.makedirs(output_dir)

# Create a new image with white background
width = 200
height = 200
img = Image.new('RGB', (width, height), 'white')
draw = ImageDraw.Draw(img)

# Draw circle border
border_color = '#1f2937'
draw.ellipse([5, 5, 195, 195], outline=border_color, width=3)

# Draw text (SYH.CLEANING)
try:
    # Try to use a better font if available
    font = ImageFont.truetype("arial.ttf", 18)
except:
    font = ImageFont.load_default()

text = "SYH.CLEANING"
bbox = draw.textbbox((0, 0), text, font=font)
text_width = bbox[2] - bbox[0]
text_x = (width - text_width) // 2
draw.text((text_x, 150), text, fill=border_color, font=font)

# Draw shoe icon (simple representation)
shoe_color = '#f3f4f6'
border = '#1f2937'
# Left shoe
draw.ellipse([40, 50, 70, 85], fill=shoe_color, outline=border, width=2)
draw.line([40, 75, 70, 75], fill=border, width=2)

# Right shoe
draw.ellipse([80, 50, 110, 85], fill=shoe_color, outline=border, width=2)
draw.line([80, 75, 110, 75], fill=border, width=2)

# Star
star_x, star_y = 120, 60
draw.polygon([
    (star_x, star_y-8),
    (star_x+3, star_y-2),
    (star_x+9, star_y-2),
    (star_x+4, star_y+2),
    (star_x+6, star_y+8),
    (star_x, star_y+4),
    (star_x-6, star_y+8),
    (star_x-4, star_y+2),
    (star_x-9, star_y-2),
    (star_x-3, star_y-2)
], fill='#3b82f6', outline='#1f2937')

# Save image
output_path = os.path.join(output_dir, 'logo.png')
img.save(output_path, 'PNG')
print(f"Logo created successfully: {output_path}")
